#! /bin/sh /usr/share/dpatch/dpatch-run
## ../makefile-CC.patch.dpatch by Jari Aalto <jari.aalto@cante.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: - Use  in Makefile

@DPATCH@

=== modified file 'wmitime/Makefile'
--- old/wmitime/Makefile	1999-03-29 03:51:39 +0000
+++ new/wmitime/Makefile	2007-04-14 15:08:02 +0000
@@ -2,17 +2,17 @@
 LIBDIR = -L/usr/X11R6/lib
 LIBS   = -lXpm -lXext -lX11 -lm
 FLAGS = -O2
+CC     = gcc
 OBJS =	wmitime.o \
 		../wmgeneral/wmgeneral.o \
 		../wmgeneral/misc.o \
 		../wmgeneral/list.o
 
-
 .c.o:
-	cc -I/usr/X11R6/share/include $(FLAGS) -D$(LANG) -c -Wall $< -o $*.o
+	$(CC) -I/usr/X11R6/share/include $(FLAGS) -c -Wall $< -o $*.o
 
 wmtime: $(OBJS)
-	cc $(FLAGS) -D$(LANG) -o wmitime $^ -lXext $(LIBDIR) $(LIBS)
+	$(CC) $(FLAGS) -o wmitime $^ -lXext $(LIBDIR) $(LIBS)
 
 all:: wmtime
 

